### PR TITLE
Add Cancel to translatable strings

### PR DESCRIPTION
**Pull request by manfred-mueller**

### CODE REVIEW

File: app/src/main/java/rkr/simplekeyboard/inputmethod/latin/settings/SettingsActivity.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public void onClick(DialogInterface dialog, int id) {
                     dialog.dismiss();
                 }
             });
-            builder.setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
+            builder.setNegativeButton(R.string.setup_cancel, new DialogInterface.OnClickListener() {
```
Comment (rkkr) [inline app/src/main/java/rkr/simplekeyboard/inputmethod/latin/settings/SettingsActivity.java:61]: `"OK"` should also be translatable in such case.

Comment (manfred-mueller) [reply app/src/main/java/rkr/simplekeyboard/inputmethod/latin/settings/SettingsActivity.java:61]: It is your decision but I'm never translating 'OK'.
I wouldn't even know how ;)

**Comment by wittmane** [comment]
Why not just use `android.R.string.cancel` (and `android.R.string.ok`) so we don't have to handle translation for something Android already does? For the context of an ok/cancel dialog, reusing the existing Android strings seems reasonable.

**Comment by manfred-mueller** [comment]
Imho you are perfectly right !
...and it is so obvious!

**Pull request closed without merging**